k2/python/k2/fsa.py

```diff
@@ -26,6 +26,57 @@ class Fsa(object):
     a vector of FSAs it is a tuple with three
     elements ``(num_fsas, None, None)``.  (Caution: it's possible
     for a vector of FSAs to have zero or one elements).
+
+    An instance of FSA has the following attributes:
+
+    - ``arcs``: You will NOT use it directly in Python. It is an instance
+                of ``_k2.RaggedArc`` with only one method ``values()`` which
+                returns a 2-D `torch.Tensor`` of dtype ``torch.int32`` with 4
+                columns. Its number of rows indicates the number of arcs in the
+                FSA. The first column represents the source states, second
+                column the destination states, third column the labels and the
+                fourth column is the score. Note that the score is actually
+                a float number but it is **reinterpreted** as an integer.
+
+    - ``score``: A 1-D ``torch.Tensor`` of dtype ``torch.float32``. It has
+                 as many entries as the number of arcs representing the score
+                 of every arc.
+
+    - ``labels``: A 1-D ``torch.Tensor`` of dtype ``torch.int32``. It has as
```

Comment:
just a question, so now the `symbol` in `Arc` definition in C++ is actually a `label`?

https://github.com/k2-fsa/k2/blob/6acfe3d54140889a8456a4002cb393607c165c64/k2/csrc/fsa.h#L24-L28

Reply:
Sorry for the confusion. `labels` is used interchangeably with the `symbol`.

Reply:
I think we can actually rename `symbol` in C++ to label, for consistency, and reserve `symbol` for the actual symbol table.